bundles/XliffBundle/src/ExportDataExtractorService/DataExtractor/DocumentDataExtractor.php

```diff
@@ -61,7 +61,7 @@ public function extract(TranslationItem $translationItem, string $sourceLanguage
      *
      * @throws \Exception
      */
-    protected function addDoumentEditables(Document $document, AttributeSet $result): DocumentDataExtractor
```

Comment:
can you please also add BC layer, so that `addDoumentEditables` returns `addDocumentEditables`, deprecation since 11.2 (we are releasing 11.1 right now, not sure would make on time) and removal in Pimcore 12.

Reply:
done :)